xikolo-ios/Views/CourseCell.swift

```diff
@@ -23,7 +25,24 @@ class CourseCell : UICollectionViewCell {
 
         nameLabel.text = course.title
         teacherLabel.text = course.teachers
-        dateLabel.text = course.language
+        dateLabel.text = course.language_translated
+
+        switch course.status {
+        case "active"?:
+            statusView.hidden = false
+            statusLabel.text = NSLocalizedString("running", comment: "course-status")
+            statusView.backgroundColor = UIColor.init(red: 140/255, green: 179/255, blue: 13/255, alpha: 1.0)
```

Comment:
Can you create a generic Brand file like in [62df1e4](https://github.com/openHPI/xikolo-ios/blob/62df1e4/xikolo-common/Branding/Brand%2Bgeneric.swift) on my branch and put these colors there?